platforms/core-runtime/launcher/src/integTest/groovy/org/gradle/launcher/SystemClassLoaderTest.groovy

```diff
@@ -48,7 +48,7 @@ class SystemClassLoaderTest extends AbstractIntegrationSpec {
                 doLast {
                     def systemLoader = ClassLoader.systemClassLoader
 
-                    systemLoader.loadClass(org.gradle.launcher.GradleMain.name) // this should be on the classpath, it's from the launcher package
+                    systemLoader.loadClass("org.gradle.launcher.daemon.bootstrap.GradleDaemon") // this should be on the classpath, it's from the launcher package
```

Comment:
`GradleMain` is no longer included in the daemon classpath, as it's only used by the client.